multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/madara/MadaraGenerator.kt

```diff
@@ -413,6 +413,7 @@ class MadaraGenerator : ThemeSourceGenerator {
         SingleLang("ReadFreeComics", "https://readfreecomics.com", "en"),
         SingleLang("ReadMangaFree", "https://readmangafree.net", "en", isNsfw = true),
         SingleLang("ReadManhua", "https://readmanhua.net", "en", overrideVersionCode = 2),
+        SingleLang("Reset Scans", "https://reset-scans.us", "en", overrideVersionCode = 1),
```

Comment:
```suggestion
        SingleLang("Reset Scans", "https://reset-scans.us", "en"),
```

The initial override version code should be 0 and omitted.

Note: it was 24 + 5 = 29 when removed, currently it's 32 + 0 = 32.